db/schema.rb

```diff
@@ -466,30 +466,30 @@
     t.float "stdev", limit: 24
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["background_id", "count_type", "tax_level", "tax_id"], name: "index_taxon_summaries_detailed", unique: true
+    t.text "rpm_list"
+    t.index %w[background_id count_type tax_level tax_id], name: "index_taxon_summaries_detailed", unique: true
     t.index ["background_id"], name: "index_taxon_summaries_on_background_id"
```

Comment:
I think with the index you added, we can and should remove the one on just background_id

Reply:
I didn't ass any index. 
the ["background_id"]  index is completely redundant. MySQL has the prefix index that having both [background_id count_type tax_level tax_id] and [background_id]   is completely unnecessary.

Reply:
agreed. Oh yeah I just saw that line 470 is in fact not new.